samples/matter/thermostat/sample.yaml

```diff
@@ -8,10 +8,9 @@ tests:
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf5340dk/nrf5340/cpuapp
-      - nrf7002dk/nrf5340/cpuapp
```

Comment:
Should we add it back if the LTO has been enabled? by default?

Reply:
Yes, I reverted the change.